front/src/applications/operationalStudies/hooks/useEtcsBrakingCurves.ts

```diff
@@ -0,0 +1,101 @@
+import { useCallback, useEffect, useMemo, useState } from 'react';
+
+import {
+  type EtcsBrakingCurve,
+  type EtcsBrakingCurves,
+  EtcsBrakingCurveType,
+  EtcsBrakingType,
+} from '@osrd-project/ui-charts';
+import { useSelector } from 'react-redux';
+
+import { useScenarioContext } from './useScenarioContext';
+import {
+  type EtcsBrakingCurvesResponse,
+  type EtcsCurves,
+  osrdEditoastApi,
+  type SimulationResponseSuccess,
+} from '../../../common/api/osrdEditoastApi';
```

Comment:
You can remove the `../../..` ;) (the reference folder is `src`)
```suggestion
} from 'common/api/osrdEditoastApi';
```

Reply:
Fixed in https://github.com/OpenRailAssociation/osrd/pull/13774/commits/d80214836de21f41ea47c9202b665d742810e027.